Sources/swift-doc/Subcommands/Generate.swift

```diff
@@ -75,6 +78,17 @@ extension SwiftDoc {
           }
         }
 
+        // Extensions on external types.
+        var symbolsByExternalType: [String: [Symbol]] = [:]
+        for symbol in module.interface.symbols.filter(symbolFilter) {
+          guard let ext = symbol.context.first as? Extension, symbol.context.count == 1 else { continue }
+          guard !declaredTypeNames.contains(ext.extendedType) else { continue }
```

Comment:
It looks like this approach is generating false positives. I tried running this against our [SwiftMarkup](https://github.com/SwiftDocOrg/SwiftMarkup) project, and it showed `Callout.Delimiter` as an extended type. 
This code appears to be picking up the conformance to `Codable` and `RawValueRepresentable` adopted in an extension in the same file.

I haven't dug into this too deeply, but my guess is that this has to do with a difference between base and qualified symbol names.

Reply:
My new approach is only _slightly_ crazy. But it has surprisingly good results for a tool that does not understand semantics.